tests/tests_unit/test_data_classes/test_capabilities.py

```diff
@@ -391,6 +393,17 @@ def test_raw_acl_database_scope(self, cognite_client, extra_existing):
             assert RawAcl([RawAcl.Action.Read], RawAcl.Scope.Table({"db1": ["t3"]})) in missing
             assert RawAcl([RawAcl.Action.Write], RawAcl.Scope.Table({"db1": ["t1"]})) in missing
 
+    def test_unknown_existing_capability(self, cognite_client):
```

Comment:
Let's also add a test using legacy cap. in a comparison (either in here or separately).